docker-compose.test.watch.yaml

```diff
@@ -1,6 +1,6 @@
 services:
   ffc-sfd-messages-processor:
     command: >
-      sh -c "npm run test:watch"
+      sh -c "git config --global --add safe.directory /home/node && npm run test:watch"
```

Comment:
**UPDATE**: Discussed with Simon and agreed that since it's just myself and Simon working on the messages services it's best to leave this rather than keep changing it every time we merge into main. It can be changed back when we move on from messages to another feature e.g. payments.